docs/ci-cd-environment/setup-caching-on-aws-s3.md

```diff
@@ -0,0 +1,164 @@
+---
+description: This guide describes how to setup the cache CLI to work with an AWS S3 bucket
+---
+
+# Setting up caching with AWS S3
+
+In order to use S3 as a storage backend, the cache CLI needs access to your AWS account. More specifically, it needs access only to perform a few actions on one S3 bucket in your AWS account.
```

Comment:
Since a clarification has been made. Why not make the sentence clearer from the beginning:

In order to use S3 as a storage backend, the cache CLI needs access only to perform a few actions on an S3 bucket in your AWS account.

I also changed `one` to `an` bucket because you may need access to more than one. Or maybe `the` bucket is best applied here. idk.

Reply:
Yeah, that sounds better 🙏